holoviews/tests/operation/test_statsoperations.py

```diff
@@ -61,7 +61,7 @@ def test_bivariate_kde_contours(self):
         kde = bivariate_kde(bivariate, n_samples=100, x_range=(0, 1),
                             y_range=(0, 1), contours=True, levels=10)
         self.assertIsInstance(kde, Contours)
-        self.assertEqual(len(kde.data), 9)
+        self.assertEqual(len(kde.data), 11)
```

Comment:
Curious of why this value is changed?

Reply:
Me too. I don't know why, I haven't looked into the KDE code. On `main` the contour levels used are
```python
[0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0]
```
and on this branch they are
```python
[0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0, 1.1]
```
Given that the data is in the range 0 to 1, the new values are better than the old I suppose.